cockroachcloud/quickstart.md

```diff
@@ -51,25 +51,27 @@ If you have not done so already, install the CockroachDB binary:
     
 <section class="filter-content" markdown="1" data-scope="linux">
 
-1. In the command line interface, use [Homebrew](https://brew.sh/) to install CockroachDB by running the following command:
+1. In your terminal, run the following command to download and extract the CockroachDB {{ page.release_info.version }} archive for Linux:
+
+    <div class="copy-clipboard">
```

Comment:
I'm a little confused about why this step isn't using the typical Markdown codeblock section.
```
    {% include_cached copy-clipboard.html %}
    ~~~ shell
    wget -qO- https://binaries.cockroachdb.com/cockroach-{{page.release_info.version}}.linux-amd64.tgz | tar xvz
    ~~~
```

Reply:
I was under the impression that this was the solution to Linux & Windows shell highlighting differences but I'm not actually sure if that's true. I may have misinterpreted the reason it was being used on the main install CockroachDB page.

Reply:
Actually, no, @ianjevans's right. I think this stuff was copied over from the install page, where we use html so that Marketing has tracking attributes in place. But on this page, we can use just straight Markdown.